pom.xml

```diff
@@ -147,6 +147,7 @@
                             <Implementation-Title>metadata-extractor</Implementation-Title>
                             <Implementation-Vendor>Drew Noakes</Implementation-Vendor>
                             <Implementation-Version>${project.version}</Implementation-Version>
+                            <Automatic-Module-Name>com.drew.imaging</Automatic-Module-Name>
```

Comment:
I think `com.drew.metadata` fits better to the artifactId and the fact that this library also supports video and audio formats.

Reply:
I'm completely open to changing it to `com.drew.metadata`.